03-make/make_homework/Makefile

```diff
@@ -0,0 +1,14 @@
+TXT:=cat #computes variable value once and just stores it
+TARGET?=sup.txt #if the variable was not defined
+
+all : stol.o
```

Comment:
Does it work as expected?

Reply:
Yes, it works as I expected, I checked and added ".log" file to the last commit. If this not as you expected could you please send me the right console output and I'll refactor this.